tests/unit/docker/test_matrix_converter.py

```diff
@@ -93,6 +175,28 @@ def test__load_cell_table_slice(self, mock_open, mock_pd_read_csv):
         self.assertIn("project.project_core.project_short_name", pandas_kwargs["names"])
         self.assertTrue(pandas_args[0].startswith("s3://"))
 
+    @mock.patch("s3fs.S3FileSystem.open")
+    def test__load_expression_table_slice(self, mock_open):
+        manifest_file_path = "tests/functional/res/expression_metadata_manifest"
+        with open(manifest_file_path) as f:
+            mock_open.return_value = f
+            self.matrix_converter.expression_manifest = self.matrix_converter._parse_manifest("test_manifest_key")
+        results = self.matrix_converter._load_expression_table_slice(0)
+        mock_open.assert_called_once()
+        self.assertEqual(type(results).__name__, 'generator')
```

Comment:
Does it make sense here to verify some values in the generator?

Reply:
hmm I think its fine the way it is